src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFiles.tsx

```diff
@@ -0,0 +1,117 @@
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  renderCreateTablesButton?: ReactNode;
+  onSectionOpenEvent?: (accordionSection: {
+    id: string;
+    title: string;
+  }) => void;
+  release: Release;
+  renderDownloadLink: (file: FileInfo) => ReactNode;
+  renderMetaGuidanceLink: ReactNode;
+  renderPreReleaseAccessLink?: ReactNode;
+}
+const ReleaseDataAndFiles = ({
```

Comment:
Might be better to use more explicit naming like `ReleaseDataAndFilesAccordion` we could reasonably expect there to be other components around data and files.